docs/rules/space-before-blocks.md

```diff
@@ -76,7 +76,7 @@ try {} catch(a) {}
 
 ### "never"
 
-The following patterns are considered problems:
+Examples of **incorrect** code for this rule with the "never" option.
```

Comment:
You end with a colon on line 30 and a period here. I like the colon better, mind switching?